wordpress-mysql-replication/README.md

```diff
@@ -1,6 +1,6 @@
 # Deploys a WordPress web site backed by MySQL master-slave replication
 
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2Fwordpress-mysql-replication%2Fazuredeploy.json" target="_blank">
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fdev%2Fwordpress-mysql-replication%2Fazuredeploy.json" target="_blank">
```

Comment:
Will make the change today ..
On Jun 22, 2016 10:23 AM, "Kay Singh" <notifications@github.com> wrote:

> In wordpress-mysql-replication/README.md
> <https://github.com/Azure/azure-quickstart-templates/pull/2145#discussion_r68096325>
> :
>
> > @@ -1,6 +1,6 @@
> >  # Deploys a WordPress web site backed by MySQL master-slave replication
> >
> > -<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2Fwordpress-mysql-replication%2Fazuredeploy.json" target="_blank">
> > +<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fdev%2Fwordpress-mysql-replication%2Fazuredeploy.json" target="_blank">
>
> Needs to be master branch
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Azure/azure-quickstart-templates/pull/2145/files/dbd332511a1d82d10acb69cecea953bdd8e4264d#r68096325>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGL5sVwYFBnU2zcUKZFfMOV7ZI2qdvVYks5qOW-qgaJpZM4I7Duf>
> .
>